decisions/2019/11/11-tapps-meeting-remote-errors-etc.md

```diff
@@ -0,0 +1,130 @@
+# 2019-11-11 @dhis2/team-apps Meeting
+
+## Attending
+
+Viktor, Austin, Jan-Gerke, Ismay, Hendrik, Birk, Erik
+
+## Maintenance App Refactor
+
+We are ready to work on it and have started prototyping: https://github.com/dhis2/mar-y/
+
+Not currently a priority, but it is something we need to spend time on when we
+can.
+
+Getting it up to a technical standard we can maintain is the first priority,
+switching out UI components to the DHIS2 ones.
+
+When it comes to improvements and functional changes to the Maintenance
+application, it is a bit stickier, as any changes to the Maintenance
+application triggers a lot of work related to updating training materials and
+training for people. We should gather requirements and write specs for changes
+we think would be inline with our way of work.
+
+Input from the field from Austin:
+
+Import/Export app is used to configure DHIS2 instead of the Maintenance
+app, since the Maintenance app does not support selecting/creating or
+uploading polygons for organisation units; only manually entered
+coordinates are supported.
+
+## Error reporting
+
+Related issues:
+
+- https://github.com/dhis2/notes/issues/76
+- https://jira.dhis2.org/browse/DHIS2-5400
+- https://jira.dhis2.org/browse/DHIS2-3481
+
+What we want to do is:
+
+- Extend the functionality of the Usage Analytics app
+
+- Build a library so that applications can report various metrics
+  that get visualised in the usage analytics app.
+
+- Extend the APIs for the usage analytics to enable additional
+  metrics (requires backend work).
+
+- Collect the data at the instance level and iteratively add more
+  features. "Send data to DHIS2 Core Team" can be a later feature.
+
+Next step is to schedule a meeting to set up a proposal to anchor the
```

Comment:
> Next step is to schedule a meeting to set up a proposal to anchor the
discussion across teams in a context.

A more concrete step than meeting-to-propose-to-discuss might be nice , like "(1) someone (@Mohammer5 or myself?) creates a proposal for tasks and timeline, (2) we meet to approve or modify it"

Reply:
Agreed. 

OT: @amcgee what do you think about us meeting and creating the draft by discussing ideas? We could just prepare ourselves and join in a hangouts call and discuss it & create a proper proposal? I guess you're more familiar with dhis2 and the already existing analytics stuff, I might add some stuff from my new relic experience and from what I was thinking about so far.

Reply:
Sounds great to me

Reply:
Sounds like you already scheduled the meeting then, I'll update the minute and merge it. Thanks guys.